qbistListeners.js

```
@@ -55,21 +61,73 @@ export function loadStateFromParam(stateBase64) {
+      console.error("Error importing file:", err)
+      alert("Error importing file")
+    }
+  }
+  input.click()
+}
+
+// Use event delegation for preview clicks since canvases get recreated
+document.getElementById("grid").addEventListener("click", (event) => {
+  const canvas = event.target.closest(".preview")
+  if (canvas) {
     const index = parseInt(canvas.id.replace("preview", ""))
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)